packages/hdwallet-walletconnectV2/src/ethereum.ts

```diff
@@ -87,11 +86,9 @@ export async function ethSignMessage(
   args: { data: string | Bytes; fromAddress: string },
   provider: EthereumProvider
 ): Promise<ETHSignedMessage | null> {
-  const buffer = isBytes(args.data) ? Buffer.from(arrayify(args.data)) : Buffer.from(args.data);
-
```

Comment:
Assuming this now comes as an hex string, we may want to use https://docs.ethers.org/v5/api/utils/bytes/#utils-isHexString to guard at runtime against non-hexlified strings being passed